app/src/main/java/com/android/voyageur/MainActivity.kt

```diff
@@ -32,7 +43,23 @@ class MainActivity : ComponentActivity() {
         Surface(
             modifier = Modifier.fillMaxSize().semantics { testTag = C.Tag.main_screen_container },
             color = MaterialTheme.colorScheme.background) {
-              VoyageurApp(placesClient)
+              val status by connectivityState()
+              val isConnected = status === ConnectionState.Available
+              Column {
+                Row {
+                  if (!isConnected) {
+                    // display a red banner at the top of the screen
+                    Text(
+                        "No internet connection",
```

Comment:
I think here it would be better to replace the string text with a string resource :)